lightning/src/ln/channelmanager.rs

```diff
@@ -6901,38 +7020,41 @@ where
 
 								log_trace!(logger, "Completing monitor update completion action for channel {} as claim was redundant: {:?}",
 									chan_id, action);
-								let (node_id, _funding_outpoint, channel_id, blocker) =
 								if let MonitorUpdateCompletionAction::FreeOtherChannelImmediately {
 									downstream_counterparty_node_id: node_id,
-									downstream_funding_outpoint: funding_outpoint,
+									downstream_funding_outpoint: _,
 									blocking_action: blocker, downstream_channel_id: channel_id,
 								} = action {
```

Comment:
I'm not sure if this is reachable, and it's pre-existing, but FYI we don't have coverage for getting a `MonitorUpdateCompletionAction::FreeOtherChannelImmediately` while `during_init`. If it's unreachable, adding a `debug_assert` could be useful.